REPOSITORIES.md

```diff
@@ -1,8 +1,8 @@
 # Repositories
 
-This document lists all repositories, along with their scope and status, within [Cilium](https://github.com/cilium).
+This document lists all non-archived repositories, sorted by their scope, within [Cilium](https://github.com/cilium).
```

Comment:
Maybe we should also have an archived section so repos can move there too

Reply:
We can do that. In that case, Archived would act as a scope, so a request for archival could be merged with the 'Change of scope' section.